toolkit/tools/pkg/imagecustomizerlib/customizepackages_test.go

```diff
@@ -211,3 +217,54 @@ func TestCustomizeImagePackagesDiskSpace(t *testing.T) {
 	assert.ErrorContains(t, err, "failed to customize raw image")
 	assert.ErrorContains(t, err, "failed to install package (gcc)")
 }
+
+func logChrootDirectoryTest(imageConnection *ImageConnection, dirPath string) ([]string, error) {
+	var folderNames []string
+
+	// Join the chroot root directory with the directory you want to log
+	fullPath := filepath.Join(imageConnection.Chroot().RootDir(), dirPath)
+
+	// Read the directory contents
+	entries, err := os.ReadDir(fullPath)
+	if err != nil {
+		logger.Log.Infof("Failed to read directory (%s): %v", fullPath, err)
+		return folderNames, fmt.Errorf("Failed to read directory (%s): %v", fullPath, err)
+	}
+
+	// Log each entry (file or directory)
+	logger.Log.Infof("Contents of directory: %s", fullPath)
+	for _, entry := range entries {
+		folderNames = append(folderNames, entry.Name())
+		logger.Log.Infof(" - %s", entry.Name())
+	}
+
+	return folderNames, nil
+}
+
+func ensureTdnfCacheCleanup(t *testing.T, imageConnection *ImageConnection, dirPath string) error {
+
+	folderNames, err := logChrootDirectoryTest(imageConnection, "/var/cache/tdnf")
+	if err != nil {
+		return err
+	}
+
+	// Ensure there are exactly 2 folders after the tdnf cleanup
+	assert.Equal(t, 2, len(folderNames), "Expected exactly 2 folders, but got %d", len(folderNames))
```

Comment:
Are these two directories empty? If they are, if might be more interesting to count the total number of files under `/var/cache/tdnf` and make sure it is 0.

You can use [filepath.Walk](https://pkg.go.dev/path/filepath#Walk) to iterate over all the files and directories. (Which also might be useful for logging.)